src/test/java/seedu/address/testutil/TypicalModules.java

```diff
@@ -101,35 +109,57 @@ public class TypicalModules {
             .build();
     //Add more
 
+
     private TypicalModules() {}
 
     public static ModulePlan getTypicalModulePlan() {
         ModulePlan mp = new ModulePlan();
 
-//        mp.addSemester(new ModulePlanSemester(new Year("1"), new Semester("1")));
-//        mp.addSemester(new ModulePlanSemester(new Year("1"), new Semester("2")));
-//        mp.addSemester(new ModulePlanSemester(new Year("2"), new Semester("1")));
-//        mp.addSemester(new ModulePlanSemester(new Year("2"), new Semester("2")));
 
         for (Module module : getTypicalModules()) {
             mp.addModule(module);
         }
         return mp;
     }
 
-    public static ModuleData getTypicalModuleData() {
+    public static ModuleData getTypicalModuleDataFromDB() {
         ModuleData moduleData = new ModuleData();
 
         for (Module m : getTypicalModules()) {
-            DbModule dbModule = new DbModule(
+            System.out.println(m);
+            Module module = new Module(
                     m.getModuleCode(), m.getName(), m.getDescription(), m.getModularCredit());
-            moduleData.addDbModule(dbModule);
+            moduleData.addModule(module);
+        }
+        return moduleData;
+    }
+    public static ModuleData getTypicalModuleData() {
+        ModuleData moduleData = new ModuleData();
+
+        for (Module m : getTypicalModules()) {
+            Module module = new Module(
+                    m.getModuleCode(), m.getYearTaken(), m.getSemesterTaken(),
+                    m.getGrade(), m.getName(), m.getDescription(), m.getModularCredit());
+            moduleData.addModule(module);
         }
```

Comment:
Undeleted print statement.

Shouldn't need two getTypicalModuleData methods. Modules in ModuleData should not have year, semester and grade fields set.